### DISCUSSION TITLE
Repo to download all plugins in 1 click

### DISCUSSION THREAD

**Discussion by alessandro-ooo** [qbittorrent/search-plugins#299] [General]
A year ago I wanted to download every plugin and install them in my client, though I found the whole process very slow and repetitive. That's why I've put every plugin in one repo, and you download them all in one click.

https://github.com/alessandro-ooo/one-click-qbittorrent-searchplugins

Last update is one year ago but you can request to add newer engines or update deprecated ones. 

IM NOT THE AUTHOR OF THESE PLUGINS, I'M ONLY MAKING IT EASY TO DOWNLOAD EVERYTHING.

**Comment by 444995** [comment]
An idea for you is to make some code that automatically updates your own repo whenever a new change is made at https://github.com/qbittorrent/search-plugins/wiki/Unofficial-search-plugins :)

**Reply by alessandro-ooo** [reply]
That's a good idea but I don't know how to do it :D

**Reply by Ouwerkerk78** [reply]
Is there any up to date ALL in one for 2025?

All older ones wont install as they are out of date

**Reply by amwink** [reply]
you can do something like<br> `wget -nc -O plugins.html https://github.com/qbittorrent/search-plugins/wiki/Unofficial-search-plugins &&`<br> `wget -nc $(grep '\.py' plugins.html | sed -e 's|py.*|py|' | sed -e 's|.*ref="||')`

**Reply by drexxdk** [reply]
I just made it https://drexxdk.github.io/qbittorrent-search-plugin-bundle/ 
Great idea

**Reply by p060477** [reply]
> I just made it https://drexxdk.github.io/qbittorrent-search-plugin-bundle/ Great idea

hi, first thxs so much @drexxdk 
i'm trying to follow the instructions:
"GitHub UI: open the repository Actions tab, select Build And Deploy, then click Run workflow."
and i am in the build and deploy but how to "run workflow" ?
i do not see the button to click
i attached the screenshoot:

<img width="1920" height="1080" alt="Immagine" src="https://github.com/user-attachments/assets/66937be5-ff6b-4468-ab41-f382fd99c924" />


thxs for yr attn and helping
cheers

**Reply by drexxdk** [reply]
Those instructions are if you want to host the solution yourself. 
You dont have to do that. If you go to https://drexxdk.github.io/qbittorrent-search-plugin-bundle/ there is a "Download bundle" button that will download the up-to-date zip

**Reply by p060477** [reply]
> Those instructions are if you want to host the solution yourself. You dont have to do that. If you go to https://drexxdk.github.io/qbittorrent-search-plugin-bundle/ there is a "Download bundle" button that will download the up-to-date zip

@drexxdk thxs so much for having clarify!
just downloaded, i see the 2 folder:privare and public
i'm interestend in public i see there are 68 files
and i've added them to my qbit 5.2.1
but how to have always them updated?
i mean what asked also by Ouwerkerk78 on his post of 2nd april 2025?
thxs so much indeed in adv
cheers

**Reply by drexxdk** [reply]
I made a Github Action that run automatically once a day that checks if the wiki page has changed. if it detects no changes then it just stops there. If it detects changes then it updates the zip file and what it shows on the website.

When it does it automatically it looks like this:
<img width="382" height="136" alt="image" src="https://github.com/user-attachments/assets/5569d540-7fa8-433c-a0e3-06e1c2017689" />

When I manually makes it run it looks like this:
<img width="473" height="135" alt="image" src="https://github.com/user-attachments/assets/9e124b86-ce1f-4172-b3b4-54bb671c05fb" />

**Reply by p060477** [reply]
@drexxdk thxs so much again
apologize my very few skills and big ignorance...but
i pray you to explain to me as you were talking to a little child and step by step:
after downloading yr zip file and installed the 68 py files included in the public subfolder
what have i to do to have all of them always updated?
i re-apologize and beg yr pardon for my big ignorance
thxs in adv for yr attn and patience too
cheers

**Reply by drexxdk** [reply]
No worries.

In the window in qBittorrent where you installed the search plugins, there is a button to sync updates to all the plugins you currently have.

To get missing plugins, you can download a new zip and drag the .py files from it into the window in qBittorrent where you install the search plugins. It will show a popup for plugins you already have installed, which can just be ignored.

**Comment by p060477** [comment]
@drexxdk  thxs so much for yr kind patience
so,apologize if i've misunderstood  due to my ignorance, the zip file to download here:
https://drexxdk.github.io/qbittorrent-search-plugin-bundle/
is day by day updated?
and there will be added also new plugins in the future so for ex the subfolder public should have more then the actual 68 py files
i've downloaded?
if yes i think the better and most useful things to do is ,when remembered, visit again that page and dl the new updated zip file
and then do this yr instruction:
"To get missing plugins, you can download a new zip and drag the .py files from it into the window in qBittorrent where you install the search plugins. It will show a popup for plugins you already have installed, which can just be ignored."
Am i wrong?
Have i misunderstood?
Apologize me again
thxs so much indeed in adv
cheers

**Reply by drexxdk** [reply]
You are welcome.

It checks for changes on the wiki with search plugins daily and if it detects any changes then it will update the zip available here https://drexxdk.github.io/qbittorrent-search-plugin-bundle/ 

It does contain every plugin available on the website that exist. The wiki links to 7 plugins that dont exist.

The zip will be up-to-date with up to a day delay. It wont update the zip or page unless the wiki changes, since that would not change anything.

It should not be something you would need to update that often. I would likely only use it on new machines where I have just installed qBittorrent and it dont have any search plugins. Other than that, maybe update it once a year.

**Comment by p060477** [comment]
@drexxdk thxs so much again for yr precious work and the time you dedicate to us!
i wish you all the best!
cheers

**Comment by p060477** [comment]
@drexxdk apologize if i'm stealing again yr precious time
looking at my qbit log i see lots of errors
i attached here the file
when you have time and if you want could you pls take a look at it and see if i have something wrong due to my big ignorance
thxs again for yr very precious and gentle attention
and big patience too
cheers
[Log.rtf](https://github.com/user-attachments/files/28346783/Log.rtf)

**Comment by p060477** [comment]
@drexxdk  to sum up the problems should be with these 13 search plugins:

`
yggtracker.py     *

https://pirateiro.com//favicon.ico    *

dodi_repacks.py      *

dontorrent.py         *

esmeraldatorrent.py    *

anidex.py        *

naranjatorrent.py      *

onlinefix.py          *

smallgames.py         *

subsplease.py          *

sukebei.py               *

yggtracker.py          *

torrentdownloads.py     *
`

hope it helps

thxs so much again for yr precious and helpful interest
cheers

**Reply by drexxdk** [reply]
I have validated that all of the .py files are valid, but that does not mean that they work within qBittorrent.

They could be trying to reach a host that no longer exist or be for a different version of qBittorrent or other issues.

I would just ignore that. Some might fail but most will work.

**Comment by p060477** [comment]
@drexxdk  thxs so much indeed, for yr kind attn, yr precious helping and yr big patience
have a nice day!
cheers!
